packages/xml-lsp-server/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@xml-tools/xml-lsp-server",
+  "description": "XML language server",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "repository": {
```

Comment:
a `files` section property may be needed to expose generated .js artifacts in npmjs.com
you can check this by running `yarn pack` and examining the tar file

Reply:
Don't understand what should be under "files" and why